apps/judicial-system/web/src/routes/Defender/Cases/defenderCases.graphql

```diff
@@ -8,6 +8,9 @@ query DefenderCases($input: CaseListQueryInput) {
     courtCaseNumber
     validToDate
     policeCaseNumbers
+    arraignmentDate {
```

Comment:
The cases query only returns courtDate. It is calculated for each case as either the arraignmentDate or the courtDate. If there is a courtDate on the case, then that date is used, otherwise arraignmentDate is used.

Note that in addition, postponedIndefinitelyExplanation is returned with the cases query. This field is need when the case has been postponed indefinitely. The case is still scheduled (á dagskrá), but no date is displayed, just the text "Frestað".